traffic_ops/testing/api/v5/origins_test.go

```diff
@@ -277,17 +277,18 @@ func TestOrigins(t *testing.T) {
 				"OK when VALID request": {
 					EndpointID:    GetOriginID(t, "origin2"),
 					ClientSession: TOSession,
-					RequestBody: tc.Origin{
-						Name:            util.Ptr("origin2"),
-						Cachegroup:      util.Ptr("multiOriginCachegroup"),
-						Coordinate:      util.Ptr("coordinate2"),
-						DeliveryService: util.Ptr("ds3"),
-						FQDN:            util.Ptr("originupdated.example.com"),
-						IPAddress:       util.Ptr("1.2.3.4"),
-						IP6Address:      util.Ptr("0000::1111"),
-						Port:            util.Ptr(1234),
-						Protocol:        util.Ptr("http"),
-						TenantID:        util.Ptr(GetTenantID(t, "tenant2")()),
+					RequestBody: tc.OriginV5{
+						Name:              "origin2",
+						Cachegroup:        util.Ptr("multiOriginCachegroup"),
+						Coordinate:        util.Ptr("coordinate2"),
+						DeliveryService:   "ds3",
+						DeliveryServiceID: GetDeliveryServiceId(t, "ds3")(),
```

Comment:
You shouldn't need to add this line. It shouldn't need to manually fetch the TenantID, either, but it was already doing that before you touched this file, so that's not your problem.